Matrix.SynapseInterop.Replication/SynapseReplication.cs

```diff
@@ -76,7 +76,14 @@ private void ReadLoop()
                     result.Append(Encoding.UTF8.GetString(buf, 0, read));
                 } while (stream.DataAvailable);
 
-                ProcessCommands(result.ToString());
+                try
+                {
+                    ProcessCommands(result.ToString());
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine("Failed to process command: {0}", ex);
```

Comment:
We should absolutely not be catching here unless we're going to write an `ERROR` over the line and close the connection.